releasenotes/notes/result-distributions-ea0760ac6efc7237.yaml

```
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Added `ProbDistribution` and `QuasiDistribution` needed for runtime results
+    for the circuit runner, and probably useful elsewhere.
```

```suggestion
    Added two new classes :class:`~qiskit.result.ProbDistribution` and
    :class:`~qiskit.result.QuasiDistribution` for dealing with probability
    distributions and quasiprobability distributions respectively. These objects
    both are dictionary subclasses that add additional methods for working
    with probability and quasiprobability distributions.
```

Also maybe having an example on how to use it at the end would be good too.